agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/common/NetworkFriendlyExceptions.java

```diff
@@ -94,10 +93,7 @@ public static boolean logSpecialOneTimeFriendlyException(
   }
 
   private static String getFriendlyExceptionBanner(String url) {
-    if (url.contains(DefaultEndpoints.LIVE_ENDPOINT)) {
-      return "ApplicationInsights Java Agent failed to connect to Live metric end point.";
-    }
-    return "ApplicationInsights Java Agent failed to send telemetry data.";
+    return "ApplicationInsights Java Agent failed to connect to " + url;
```

Comment:
@kryalama we lost a little bit here but I think it's ok, still gives the info needed for troubleshooting